packages/dialtone-vue2/components/rich_text_editor/extensions/mentions/MentionSuggestion.vue

```diff
@@ -9,7 +9,9 @@
       :image-alt="name"
       size="xs"
     />
-    {{ name }}
+    <span class="mention-suggestion-name">
+      {{ name }}
+    </span>
```

Comment:
I can also make this here:
```suggestion
    <span class="d-truncate">
      {{ name }}
    </span>
```
and skip the eslint warning with `<!-- eslint-disable vue/no-restricted-class -->`
Let me know what works better for you

Reply:
nah the way you did it is better. We don't want to use util classes in components because of `!important`